app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -37,15 +37,33 @@ def vmt_candidates_for(appliance)
           appliance_type: appliance.appliance_type,
           state: 'active',
         )
-        vmts.select do |vmt|
-          (vmt.tenants & appliance.tenants.active.funded_by(appliance.fund)).present?
+        if appliance.tenants.present?
+          vmts = restrict_by_user_requirements(vmts, appliance)
         end
+        vmts = restrict_by_tenant_availability(vmts, appliance)
+        vmts
       end
 
       private
 
       attr_reader :appliance
 
+      def restrict_by_user_requirements(vmts, appliance)
+        # If the user requests that the appliance be bound to a specific set of tenants,
```

Comment:
If you want to create method documentation please use rdocs syntax before method instead of adding comment inside method.